src/core/public/application/types.ts

```diff
@@ -685,6 +685,11 @@ export interface NavigateToAppOptions {
    * if true, will not create a new history entry when navigating (using `replace` instead of `push`)
    */
   replace?: boolean;
+
+  /**
+   * if true, will try to open the app in new tab, will copy session information via window.open
+   */
+  openInNewTab?: boolean;
```

Comment:
Why ` will try`? I don't see any `try / fallback to just open in current tab` logic in the implementation?

Reply:
Updated in [`c1d3f3f` (#96723)](https://github.com/elastic/kibana/pull/96723/commits/c1d3f3f0cf8e8258c02a393daa71c74ca161fc46)